src/components/Popups/ReplyField/ReplyField.tsx

```diff
@@ -69,6 +86,28 @@ export default class ReplyField extends React.Component<Props, State> {
         this.saveCursorPosition();
     }
 
+    getItemList = (editorState: EditorState): Collaborator[] => {
+        const { collaborators } = this.props;
+
+        const activeMention = getActiveMentionForEditorState(editorState);
+        if (!activeMention) {
+            return [];
+        }
+
+        const trimmedQuery = activeMention.mentionString.trim();
+
+        if (trimmedQuery.length < 2) {
+            return collaborators;
+        }
+
+        return collaborators.filter(({ item }) =>
+            item
+                ? fuzzySearch(trimmedQuery, item.name, 0) ||
+                  ('email' in item && fuzzySearch(trimmedQuery, item.email, 0))
```

Comment:
I find this difficult to read. Can we use some variables or helper methods to clear it up?